packages/components/src/util/typography.tsx

```diff
@@ -93,6 +100,11 @@ function Typography<IComponent extends React.ElementType>({
         // Weights
         weight === "bold" && styles.weightBold,
         weight === "normal" && styles.weightNormal,
+        // Spacing
+        spacing === "none" && styles.spacingNone,
```

Comment:
should this be removed now?

Reply:
I like the idea of having the ability to be explicit in the case that someone is using a condition, something like:

`spacedOut ? '1x' : 'none'` instead of `spacedOut ? '1x' : undefined`

Reply:
oh as in there's no corresponding `.spacingNone` style in typography.module.css now